Formula/grass.rb

```diff
@@ -0,0 +1,352 @@
+      "--with-readline-includes=#{Formula["readline"].opt_include}",
+      "--with-readline-libs=#{Formula["readline"].opt_lib}",
+      "--with-blas",
+      "--with-blas-includes=#{Formula["openblas"].opt_include}",
+      "--with-blas-libs=#{Formula["openblas"].opt_lib}",
+      "--with-geos=#{Formula["geos"].opt_bin}/geos-config",
+      "--with-geos-includes=#{Formula["geos"].opt_include}",
+      "--with-geos-libs=#{Formula["geos"].opt_lib}",
+      "--with-odbc",
+      "--with-odbc-includes=#{Formula["unixodbc"].opt_include}",
+      "--with-odbc-libs=#{Formula["unixodbc"].opt_lib}",
+      "--with-gdal=#{Formula["gdal"].opt_bin}/gdal-config",
+      "--with-zlib-includes=#{Formula["zlib"].opt_include}",
+      "--with-zlib-libs=#{Formula["zlib"].opt_lib}",
+      "--with-bzlib",
+      "--with-bzlib-includes=#{Formula["bzip2"].opt_include}",
+      "--with-bzlib-libs=#{Formula["bzip2"].opt_lib}",
+      "--with-cairo",
+      "--with-cairo-includes=#{Formula["cairo"].opt_include}/cairo",
+      "--with-cairo-libs=#{Formula["cairo"].opt_lib}",
+      "--with-cairo-ldflags=-lfontconfig",
+      "--with-freetype",
+      "--with-freetype-includes=#{Formula["freetype"].opt_include}/freetype2",
+      "--with-freetype-libs=#{Formula["freetype"].opt_lib}",
+      "--with-proj-includes=#{Formula["proj"].opt_include}",
+      "--with-proj-libs=#{Formula["proj"].opt_lib}",
+      "--with-proj-share=#{Formula["proj"].opt_share}/proj",
+      "--with-tiff",
+      "--with-pdal=#{Formula["pdal"].opt_bin}/pdal-config",
+      "--with-tiff-includes=#{Formula["libtiff"].opt_include}",
+      "--with-tiff-libs=#{Formula["libtiff"].opt_lib}",
+      "--with-png",
+      "--with-png-includes=#{Formula["libpng"].opt_include}",
+      "--with-png-libs=#{Formula["libpng"].opt_lib}",
+      "--with-regex",
+      # "--with-regex-includes=#{Formula["regex-opt"].opt_lib}",
+      # "--with-regex-libs=#{Formula["regex-opt"].opt_lib}",
+      "--with-fftw",
+      "--with-fftw-includes=#{Formula["fftw"].opt_include}",
+      "--with-fftw-libs=#{Formula["fftw"].opt_lib}",
+      "--with-sqlite",
+      "--with-sqlite-includes=#{Formula["sqlite"].opt_include}",
+      "--with-sqlite-libs=#{Formula["sqlite"].opt_lib}",
+      "--with-mysql",
+      "--with-mysql-includes=#{Formula["mysql"].opt_include}/mysql",
+      "--with-mysql-libs=#{Formula["mysql"].opt_lib}",
+      "--with-postgres",
+      "--with-postgres-includes=#{Formula["postgresql"].opt_include}",
+      "--with-postgres-libs=#{Formula["postgresql"].opt_lib}",
+      # Error: liblas has been disabled because it is not supported upstream!
+      # "--with-liblas",
+      # "--with-liblas-includes=#{Formula["liblas"].opt_include}",
+      # "--with-liblas-libs=#{Formula["liblas"].opt_lib}",
+      "--with-opengl=macosx",
+      "--with-opencl",
+      "--with-openmp",
+      "--with-openmp-includes=#{Formula["libomp"].opt_include}",
+      "--with-openmp-libs=#{Formula["gcc"].opt_lib}/gcc/current",
```

Comment:
It looks like you're mixing headers from one OpenMP implementation and libraries from another. You shouldn't do this.

Reply:
The thing here is that libomp that is available in Homebrew does not contain `GOMP_parallel_start` which is needed for GRASS to build with openMP support.

```
checking whether to use OpenMP... "yes"
checking for location of OpenMP includes... /opt/homebrew/opt/libomp/include
checking omp.h usability... yes
checking omp.h presence... yes
checking for omp.h... yes
checking for location of OpenMP library... /opt/homebrew/opt/libomp/lib
checking for GOMP_parallel_start... no
checking for GOMP_parallel_start in -lgomp... no
configure: error: *** Unable to locate OpenMP library.
```

As I've seen GCC includes contain libgomp. I see that `libgomp` is not available, so I thougt of using it from GCC. This isn't exactly a justification to do this, but I haven't found another way around this. I would appreciate if someone would be kind to point me as to where I could `libgomp` in an openMP implementation via homebrew that I could use as a dependency.